__tests__/notifications.ts

```diff
@@ -1465,3 +1483,173 @@ describe('streamNotificationUsers', () => {
+        pollResultAuthorNotification,
+        {
+          entityId: poll.id,
+          entityTableName: getTableName(con, PollPost),
+          scheduledAtMs: Date.now(),
+          delayMs: 1000,
+        },
+      );
+
+    expect(result).toHaveLength(1);
+    expect(result![0].type).toBe(NotificationType.PollResult);
+    expect(result![0].ctx.userIds).toEqual(['1']);
+  });
+
+  it('should send notifications to poll voters when poll expires', async () => {
+    const poll = await createPollPost('1'); // user '1' is the author
+    await createPollOptions(poll.id);
+    await createPollVotes(poll.id, ['2', '3', '4']); // users 2, 3, 4 voted
+
+    const result =
+      await invokeTypedNotificationWorker<'api.v1.delayed-notification-reminder'>(
+        pollResultNotification,
+        {
+          entityId: poll.id,
+          entityTableName: getTableName(con, PollPost),
+          scheduledAtMs: Date.now(),
+          delayMs: 1000,
+        },
+      );
+
+    expect(result).toHaveLength(1);
+    expect(result![0].type).toBe(NotificationType.PollResult);
+    expect(result![0].ctx.userIds).toEqual(['2', '3', '4']);
+  });
+
+  it('should exclude poll author from voter notifications', async () => {
+    const poll = await createPollPost('1'); // user '1' is the author
+    await createPollOptions(poll.id);
+    await createPollVotes(poll.id, ['1', '2', '3']); // author also voted
+
+    const result =
+      await invokeTypedNotificationWorker<'api.v1.delayed-notification-reminder'>(
+        pollResultNotification,
+        {
+          entityId: poll.id,
+          entityTableName: getTableName(con, PollPost),
+          scheduledAtMs: Date.now(),
+          delayMs: 1000,
+        },
+      );
+
+    expect(result).toHaveLength(1);
+    expect(result![0].type).toBe(NotificationType.PollResult);
+    expect(result![0].ctx.userIds).toEqual(['2', '3']); // author excluded
+  });
+
+  it('should return nothing if no voters exist', async () => {
+    const poll = await createPollPost('1');
+    // No votes created
+
+    const result =
+      await invokeTypedNotificationWorker<'api.v1.delayed-notification-reminder'>(
+        pollResultNotification,
+        {
+          entityId: poll.id,
+          entityTableName: getTableName(con, PollPost),
+          scheduledAtMs: Date.now(),
+          delayMs: 1000,
+        },
+      );
+
+    expect(result).toBeUndefined();
+  });
+
+  it('should return nothing if poll does not exist', async () => {
+    const result =
+      await invokeTypedNotificationWorker<'api.v1.delayed-notification-reminder'>(
+        pollResultNotification,
+        {
+          entityId: 'non-existent-poll',
+          entityTableName: getTableName(con, PollPost),
+          scheduledAtMs: Date.now(),
+          delayMs: 1000,
+        },
+      );
+
+    expect(result).toBeUndefined();
+  });
+
+  it('should cancel entity reminder workflow when poll is deleted', async () => {
```

Comment:
cdc tests got to `__tests__/workers/cdc/primary`, and also if you can add a test for `runEntityReminderWorkflow`, you can check how I did it for campaign